crates/eips/Cargo.toml

```diff
@@ -12,8 +12,12 @@ repository.workspace = true
 exclude.workspace = true
```

Comment:
this file appears to have no functional changes. can you restore it to condition on `main` ?